sktime/classification/kernel_based/tests/test_arsenal.py

```diff
@@ -6,6 +6,12 @@
 from sktime.tests.test_switch import run_test_for_class
 
 
+# A reference to this issue is also present inside sktime/tests/_config.py,
+# and needs to be removed from `EXCLUDED_TESTS` upon resolution.
+@pytest.mark.skip(
```

Comment:
this is sporadic, we should not skip it

Reply:
What's the general consensus on dealing with such sporadic errors?